src/test.js

```diff
@@ -0,0 +1,26 @@
+const { notifier, logger } = require('./entry')({
+  BUGS_TOKEN: '00000000-0000-0000-0000-000000000000',
+  LOGS_TOKEN: '00000000-0000-0000-0000-000000000000'
+});
+
+console.log(typeof notifier);
+console.log(typeof logger);
+console.log();
+console.log(logger.info);
+console.log(logger.warn);
+console.log(logger.error);
+console.log(logger.debug);
+console.log(logger.silly);
+console.log(logger.verbose);
+console.log();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.dialonce.io/coding_rules#rule_key=javascript%3AS2228)